lib/auth/grpcserver.go

```diff
@@ -768,38 +770,52 @@ func (g *GRPCServer) ClearAlertAcks(ctx context.Context, req *authpb.ClearAlertA
 	return &emptypb.Empty{}, nil
 }
 
+// GetUser returns a user matching the provided name if one exists.
+// TODO(tross): DELETE IN 16.0.0
+// Deprecated: use [usersv1.Service.GetUser] instead.
 func (g *GRPCServer) GetUser(ctx context.Context, req *authpb.GetUserRequest) (*types.UserV2, error) {
-	auth, err := g.authenticate(ctx)
+	usersService, err := usersv1.NewService(usersv1.ServiceConfig{
```

Comment:
Could we call the actual UsersService, using a local gRPC client, instead of instantiating it here? gRPC implementation types are not really meant to be used directly like this.

Reply:
This looks good, thanks for the changes.